guide/lib/helpers/content_helpers.rb

```diff
@@ -9,34 +11,42 @@ def site_title
     end
 
     def alert_info
-      {
-        "Alertes - Alerts sur la documentation du Système de Design de lʼÉtat" => "https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/alerte/"
-      }
+      dsfr_component_doc_link("Alerte")
     end
 
     def accordion_info
-      {
-        "Accordéons sur la documentation du Système de Design de lʼÉtat" => "https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/accordeon/"
-      }
+      dsfr_component_doc_link("Accordéons", "accordeon")
     end
 
     def link_info
-      {
-        "Les liens sur la documentation du Système de Design de lʼÉtat" => "https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/liens/"
-      }
+      dsfr_component_doc_link("Liens")
     end
 
     def tile_info
-      {
-        "Tuiles sur la documentation du Système de Design de lʼÉtat" => "https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/tuile/"
-      }
+      dsfr_component_doc_link("Tuile")
     end
 
     def component_helper_mapping
       {
         "DsfrComponent::AlertComponent" => "dsfr_alert",
-        "DsfrComponent::AccordionComponent" => "dsfr_accordion"
+        "DsfrComponent::AccordionComponent" => "dsfr_accordion",
+        "DsfrComponent::TileComponent" => "dsfr_tile",
+        "DsfrComponent::Badge" => "dsfr_badge"
       }
     end
+
+    def badge_info
+      dsfr_component_doc_link("Badge")
+    end
+
+  private
+
+    def dsfr_component_doc_link(name, id = nil)
```

Comment:
nice ! one step towards generization of these weird methods. thank you 🙇 

I'm guessing we'll just need a hash map in a future refactor, like `component_helper_mapping` but for the DSFR id